source/sass/patterns/molecules/tag-list.scss

```diff
@@ -11,6 +11,10 @@
   width: 100%;
 }
 
+.tag-list--reduced-standoff {
```

Comment:
Different approach to `.teaser__heading` and `.teaser__details` (which do change the spacing of another pattern).

Difference here is a molecule including another molecule, so order is based just on filename (which would work in this case, but not necessarily others).

Name here feels a little forced (though can't think of a better one). Feels like block spacing could be the one case where it's ok to override it, but the above problem remains.

🤔

Reply:
Hmmm... I think there's a difference between styling an html element that happens to be a pattern because of how we're approaching things, and styling a collection of html elements, a more conventional definition of a pattern.

Styling elements like `<p>` and `<h[n]>` within a pattern using a css class name scoped to the calling pattern makes sense to me, and will prevent potentially many variants being created for the html element patterns.

When including higher order patterns I think it makes sense to create variants for these to make it clearer where the responsibilities lie.

Reply:
Explicitly stating the overall order of pattern imports by hierarchy level should get over the problem of uncertainty through filename ordering https://github.com/libero/pattern-library/pull/119/files#diff-be3f8b4b67f796cfebba62ea9208cf19R3

Reply:
Atoms aren't the same as HTML elements (they should be a single element, but there can be multiple atoms per element). Agree if we are talking elements; are atoms being overused?

I'm fine with variants as long as they aren't tied to a specific usage, but they should still have an example.

Main problem here is a good name for it!

(Makes me think about https://github.com/libero/pattern-library/pull/66#discussion_r265019000 again too.)

Reply:
Rather than `reduced-standoff` for the shorter one used in the teaser, we could flip it: the shorter one in the teaser is the base one, and we call the one in the content header the `taller` variant?

Reply:
Feels like `tall` should be the default, so going to replace it with `compact` (rather than `short`).

Still think we should readdress this more generally, and this does help reduce the impact of the change.